auth/utils/read-external-sources.ts

```diff
@@ -16,7 +16,6 @@
 
 import camelcase = require('camelcase');
 import isEmpty = require('lodash.isempty');
-import vcapServices = require('vcap_services');
```

Comment:
If we're removing all use of this package, it should be removed from the `package.json` as well.